### ISSUE TITLE
why i just get validation top1 error,but without train top1error and validation loss?thank you very much!

### ISSUE THREAD

**Issue by mingji123** [wenxinxu/resnet_in_tensorflow#38]
like this,when i run the code in terminal ,.
it just get 
Train top1 error = 
Validation top1 error = 0.4200
Validation loss = 

so why that train top1 error and validation loss is no output

thank you very much!
